src/modules/history.js

```diff
@@ -0,0 +1,54 @@
+/**
+* history
+* @author Deux Huit Huit
+*/
+(function ($, undefined) {
+
+	'use strict';
+
+	var scope = $('#site');
+
+	var sels = {
+		input: '.js-history-input'
+	};
+
+	var urls = [];
+
+	var update = function () {
+		urls.push(window.location.href);
+		scope.find(sels.input).each(function () {
+			var t = $(this);
+			var value = urls.join('\n');
+
+			if (t.attr('data-history-unique') === 'true') {
```

Comment:
Hum... Should it be default ?

Reply:
I mean, if we remove duplicate we can't directly see the path the user did before a form submit.